Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -176,6 +183,19 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
                 errorsCount -= 1
             }
 
+        case let command as RUMAddLongTaskCommand where isActiveView:
+            longTasksCount += 1
+            if command.duration.toInt64Nanoseconds > Constants.frozenFrameThresholdInNs {
+                frozenFramesCount += 1
+            }
+
+            if sendLongTaskEvent(on: command) {
+                needsViewUpdate = true
+            } else {
+                longTasksCount -= 1
+                frozenFramesCount -= 1
```

Comment:
If `frozenFramesCount += 1` was not executed, this will result with unnecessary decrement, 🐞 no? I think we need to check `duration` once again for this decrement.